src/RepoToolset/Settings.props

```diff
@@ -1,15 +1,22 @@
-﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project>
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
   <Import Project="DefaultVersions.props"/>
   <Import Project="ProjectLayout.props" />
   <Import Project="StrongName.props"/>
   <Import Project="Version.props"/>
   <Import Project="ProjectDefaults.props"/>
-  <Import Project="Tools.props" />
   <Import Project="Tests.props" />
-  
+
+  <Import Project="Tools.props" Condition="'$(UsingToolMicrosoftNetCompilers)' != 'false'" />
```

Comment:
>Tools [](start = 19, length = 5)

Rename the file to Compilers.props, since it's now conditioned on UsingToolMicrosoftNetCompilers